runtime/lib/ttnn/operations/linear/linear.h

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TTNN_RUNTIME_LINEAR_H
+#define TTNN_RUNTIME_LINEAR_H
+
+#include "tt/runtime/ttnn/types.h"
+#include "ttmlir/Target/TTNN/program_generated.h"
+
+namespace tt::runtime::ttnn::operations::linear {
```

Comment:
We should add under the matmul folder in the matmul namespace, ttnn models this op as a matmul with bias.

Reply:
I agree with that, can you check it now? By the way, I separated namespace blocks in matmul.cpp to not interfere with existing matmul documentation, as linear doesn't bring anything new to the table in that regard.

Reply:
Great this looks good, thanks for looking into it. I think we can merge the namespaces, and just wrap the anchor around the run function of matmul instead of the entire namespace.